make-release.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+# Release process automation script.
+# Used to create branch/tag and update VERSION files
+# and and trigger release by force pushing changes to the release branch
+
+# set to 1 to actually trigger changes in the release branch
+TRIGGER_RELEASE=0
+NOCOMMIT=0
+MAJOR_VERSION_BUMP=0
+
+usage ()
+{
+  echo "Usage: $0 --repo [GIT REPO TO EDIT] --version [VERSION TO RELEASE] [OPTIONS]"
+  echo "Options:"
+  echo "    --help, -h"
+  echo "        Show this message."
+  echo "    --no-commit, -n"
+  echo "        Do not commit changes."
+  echo "    --trigger-release, -t"
+  echo "        Push changes to remote repo, triggering release."
+  echo "Example: $0 --repo git@github.com:eclipse/che-subproject --version 'v3.2.1' --trigger-release"; echo
+}
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+    '-r'|'--repo') REPO="$2"; shift 1;;
+    '-v'|'--version') VERSION="$2"; shift 1;;
+    '-t'|'--trigger-release')
+      if [[ ! ${NOCOMMIT} -eq 0 ]]; then
+        echo "Only one of --trigger-release and --no-commit may be specified"
+        exit 1
+      fi
+      TRIGGER_RELEASE=1; shift 0;;
+    '-n'|'--no-commit')
+      if [[ ! ${NOCOMMIT} -eq 0 ]]; then
```

Comment:
```suggestion
      if [[ ! ${TRIGGER_RELEASE} -eq 0 ]]; then
```
?

Reply:
Good catch, thanks.

Reply:
just a nitpick, but why do a if "not equal 0" when you can do `if [[ ${var} -gt 0 ]]` ?